pkg/types/installconfig.go

```diff
@@ -45,6 +45,8 @@ type Admin struct {
 	Email string `json:"email"`
 	// Password is the password of the admin user.
 	Password string `json:"password"`
+	// SSHKey to use for the access to compute instances.
+	SSHKey string `json:"sshKey"`
```

Comment:
why this is optional?

Reply:
> why this is optional?

Maybe the installer doesn't care about SSH access?  But mostly because it's `omitempty` [here][1], and I'd rather stay consistent ;).

[1]: https://github.com/openshift/installer/pull/127/files#diff-a72bd0dbf7b205c5c213ee3b6d7f1717R23

Reply:
ok, updated